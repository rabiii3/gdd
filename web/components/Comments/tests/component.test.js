import React from 'react';
import { shallow } from 'enzyme';
import { Comments } from '..';

describe('web | components | Comments | component', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        user: {},
        people: {},
        isEditable: "",
        setEdit: jest.fn(() => {}),
        actions: {},
        classes: {},
        comments: [],
      };
      const wrapper = shallow(<Comments {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
