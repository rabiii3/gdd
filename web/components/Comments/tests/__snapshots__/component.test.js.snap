// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | Comments | component render should match snapshot 1`] = `
<WithStyles(Grid)
  container={true}
>
  <WithStyles(Grid)
    item={true}
    xs={true}
  />
</WithStyles(Grid)>
`;
